apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -72,8 +73,21 @@ export const addStudentLevelPairing = (scriptId, studentLevelPairing) => {
 
 export const jumpToLessonDetails = lessonOfInterest => {
   return (dispatch, getState) => {
+    const state = getState().sectionProgress;
+    const currentUserState = getState().currentUser;
     dispatch(setLessonOfInterest(lessonOfInterest));
     dispatch(setCurrentView(ViewType.DETAIL));
+    firehoseClient.putRecord({
+      study: ' teacher_dashboard_actions',
+      study_group: 'progress',
+      event: 'view_change_toggle',
+      user_id: currentUserState.userId,
+      data_json: JSON.stringify({
+        section_id: state.section.id,
+        old_view: ViewType.SUMMARY,
+        new_view: ViewType.DETAIL
+      })
+    });
```

Comment:
maybe even combine here too.  Could `logToggle` live in Redux and be exported and reused in the component?